airbyte-integrations/connectors/source-mongodb-internal-poc/metadata.yaml

```diff
@@ -1,20 +0,0 @@
-data:
-  connectorSubtype: database
-  connectorType: source
-  definitionId: 5ac5a7e5-43f5-4e7a-bf53-70961b0307bc
-  dockerImageTag: 0.0.1
-  dockerRepository: airbyte/source-mongodb-internal-poc
-  githubIssueLabel: source-mongodb-internal-poc
-  icon: mongodb.svg
-  license: ELv2
-  name: MongoDb POC
-  registries:
-    cloud:
-      enabled: true
```

Comment:
One minor note here @jdpgrailsdev is that right now to remove this connector from the registry you will need to merge a metadata.yaml file with 
```yaml
  registries:
    cloud:
      enabled: false
    oss:
      enabled: true
```

Afterwards you are free to delete the file.